.github/dependabot.yml

```diff
@@ -26,6 +26,11 @@ updates:
   schedule:
     interval: "daily"
 
+- package-ecosystem: "pip"
```

Comment:
Why is this needed if always empty on main?

Reply:
i have to ~permanently add this to work on any pytooling or external python libs, so it would be v helpful for me not having to add this boilerplate every time

its come up now as im showing @kfaseela how pytooling works and can be worked on with Envoy main, and working on some dev docs

adding this will make documenting dev/ci testing with pytooling a lot easier

Reply:
rereading this, and im wondering if perhaps you meant why was the dependabot entry required

that is because we have a check for it - we could potentially adjust the check to ignore the dev requirement and remove this

Reply:
Yes, that is what I meant. Might be best to skip the check.

Reply:
sure, ill hack the `pip_check` to ignore...

Reply:
pip_check updated and the dependabot stub removed